build-tools/packages/build-cli/src/lib/github.ts

```diff
@@ -102,7 +102,6 @@ export async function createPullRequest(
 		repo: pr.repo,
 		pull_number: newPr.data.number,
 		reviewers: pr.reviewers,
-		team_reviewers: [""],
```

Comment:
@sonalideshpandemsft I think this is the change that should fix the _"HttpError: Reviews may only be requested from collaborators. One or more  of the teams you specified is not a collaborator of the  microsoft/FluidFramework repository."_ error.